cclib/parser/nwchemparser.py

```diff
@@ -480,10 +498,10 @@ def extract(self, inputfile, line):
             at_and_dashes = next(inputfile)
             line = next(inputfile)
             assert int(line.split()[1]) == self.geostep == 0
-            gmax = float(line.split()[4])

```

Comment:
Since this code is being touched, can you replace all the repeated `line.split()` calls (starting on old line 482/new line 500) with a `tokens = line.split()` and use `tokens` everywhere?